packages/foam-vscode/src/core/model/uri.ts

```diff
@@ -303,33 +205,6 @@ function percentDecode(str: string): string {
   );
 }
 
-/**
- * Converts a windows-like path to standard URI path
- * - Normalize the Windows drive letter to upper case
- * - replace \ with /
- * - always start with /
- *
- * see https://github.com/foambubble/foam/issues/813
- * see https://github.com/microsoft/vscode/issues/43959
- * see https://github.com/microsoft/vscode/issues/116298
- *
- * @param path the path to convert
- * @returns the URI compatible path
- */
-function windowsPathToUriPath(path: string): string {
```

Comment:
As I said before, uriPath is used in the sense of platform-agnostic path, as opposed to platform-specific paths like POSIX or Win32. Moreover, this clearly conveys the fact that these paths are often found in `uri.path`. It's a soft way of communicating the missing abstraction that would otherwise require a `Path` class or enlarging the `URI` API too much. I've done some further changes in that direction, but it's not possible to include all path manipulation use cases in the URI API in a way that makes complete sense. Some manipulations are intrinsically low-level and at that level paths are little more than strings, for example when resolving identifiers in the workspace. So I need functions over strings on the one hand, and a higher level abstraction at the other hand. It's not easy to find a fully satisfying middle ground.

Reply:
`uriPath` is now just `path` everywhere. This is explained in detail somewhere above or below.